src/main/scala/ai/privado/passes/PropertyParserPass.scala

```diff
@@ -354,7 +355,8 @@ class PropertyParserPass(
   }
 
   private def addFileNode(name: String, builder: BatchedUpdate.DiffGraphBuilder): NewFile = {
-    val fileNode = NewFile().name(name)
+    val relativeFileName = Path.of(projectRoot).relativize(Path.of(name)).toString
+    val fileNode         = NewFile().name(relativeFileName)
```

Comment:
Is this in sync with what joern has? We wouldn't want to be out of sync with them where only the property files have relative paths, and others don't.

Reply:
Yes, we have relative paths across

Reply:
Approved. You might also want to check if we missed any custom nodes for relative paths.